etc/kayobe/ansible/vault-deploy-overcloud.yml

```diff
@@ -43,6 +43,7 @@
       file:
         path: /opt/kayobe/vault
         state: directory
+      become: true
```

Comment:
I think that typically `/opt/kayobe` gets created during this task:
```yaml
        - name: Ensure global virtualenv directory exists
          file:
            path: "{{ virtualenv_path }}"
            state: directory
            owner: "{{ ansible_facts.user_uid }}"
            group: "{{ ansible_facts.user_gid }}"
            mode: 0755
          # Check whether the virtualenv directory is a subdirectory of the
          # global virtualenv directory.
          when: virtualenv.startswith(virtualenv_path)
          become: True
```

So it will be stack:stack 0755. I wonder what we require here? I suppose root:root 0700 would be most secure, if that works.